tests/E2E/EntryBasicTest.php

```diff
@@ -24,20 +25,43 @@ public function setUp()
     }
 
     /**
-     * @vcr e2e_entry_get_all.json
+     * @vcr e2e_entry_get_all_locale_all.json
      */
     public function testGetAll()
     {
+        $query = (new Query())
+            ->setLocale('*');
+        $assets = $this->client->getEntries($query);
+
+        $this->assertInstanceOf(ResourceArray::class, $assets);
+    }
+
+    /**
+     * @vcr e2e_entry_get_all_locale_default.json
+     */
+    public function testGetAllDefaultLocale()
+    {
         $assets = $this->client->getEntries();
 
```

Comment:
I feel like you can assert some more stuff here. Just testing that it's an array doesn't test that the data is what you expect it to be. Maybe assert that the first element in the array is a certain thing with a certain localized title etc.